src/util.rs

```diff
@@ -195,43 +210,32 @@ pub fn derive_date_from_url(url: &str) -> Result<(ProcessingType, URLData)> {
                                 Ok(f) => f,
                                 Err(e) => Err(e)?,
                             };
-                            let selector = match Selector::parse("time") {
-                                Ok(s) => s,
-                                Err(_) => Err(DeriveDateError::SelectorError)?,
-                            };
 
-                            let date_init = match fragment
-                                .select(&selector)
-                                .nth(0)
-                                .ok_or(DeriveDateError::ScraperElementError)
-                            {
-                                Ok(d) => {
-                                    match d
-                                        .value()
-                                        .attr("datetime")
-                                        .ok_or(DeriveDateError::ScraperAttributeError)
-                                    {
-                                        Ok(s) => s.to_string(),
-                                        Err(e) => return Err(e)?,
-                                    }
+                            let extracted_results: ExtractedTimestamps = match processing_type {
+                                Some(ProcessingType::Bruteforce) => {
+                                    info!("Bruteforcing for timestamps...");
```

Comment:
This applies to other added ```info!```s as well: I think they should only fire if the ```simple``` flag isn't on, just to not mess with the output in case someone uses the CLI mode.

![WindowsTerminal_2022-09-08_12-07-37](https://user-images.githubusercontent.com/41237021/189086955-6428cda8-d370-4bb4-a177-b452bdfc6e16.png)

https://github.com/vyneer/tbf/blob/main/src/twitch/clips.rs#L109 - something like this.

This means adding an additional argument to functions, could either be a bool (probably the easier way) or a Flags struct.

Reply:
On the topic of flags, I noticed now that within [src/main.rs](https://github.com/vyneer/tbf/blob/629d87f523d369476413d60f4b5e8a2ae79070b9/src/main.rs#L67) verbose and simple flags are explicitly set to false. Shouldn't we be checking to see if the user has supplied their own variable, and let it fallback on the default values as specified in src/config.rs?

Reply:
Oh, true, you're right! Should definitely change that.